index.html

```diff
@@ -1,53 +1,93 @@
-<!-- Copyright 2020 Google LLC. All Rights Reserved.
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="Mark Otto, Jacob Thornton, and Bootstrap contributors">
+    <meta name="generator" content="Jekyll v3.8.5">
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
+    <link rel="stylesheet" href="index.css">
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 
-    http://www.apache.org/licenses/LICENSE-2.0
+    <title>👨 Don't Touch Your Face 👩‍</title>
+  </head>
+  <body class="text-center">
+    <div class="cover-container d-flex w-100 h-100 p-3 mx-auto flex-column">
 
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-==============================================================================-->
+  <div class="container">
 
-<head>
-  <!-- Load three.js -->
-<script src="https://cdn.jsdelivr.net/npm/three@0.106.2/build/three.min.js"></script>
-<!-- Load scatter-gl.js -->
-<!-- Use npm package once https://github.com/PAIR-code/scatter-gl/issues/36 is fixed -->
-<script src="https://storage.googleapis.com/learnjs-data/handtrack_staging/scatter-gl.js"></script>
-<style>
-  .detector-canvas-wrapper, .detector-scatter-gl-container {
-    display: inline-block;
-    vertical-align: top;
-  }
+    <div class="jumbotron">
+      <h1>👨 Don't Touch Your Face 👩‍</h1> 
+    </div>
+    <div class="row justify-content-center">
+      <p class="justify-content-md-center col-md-8 text-left">
+        <strong>To protect yourself during the outbreak of COVID-19, this app helps you keeps your hands off your face using your webcam and machine learning.<br><br>We take your privacy very seriously. All of the camera data stays local and never leaves your computer. This app works locally in your browser.</strong>
+      </p>
+    </div>
+  </div>
+  
+  <main role="main" class="inner cover">
+    <br>
```

Comment:
Ideally, we should not be using br. Instead, padding, border, <p>, or some other bootstrap element should be used.